pxtcompiler/emitter/hexfile.ts

```diff
@@ -132,10 +132,10 @@ namespace ts.pxtc {
             }
         }
 
-        export function encodeVTPtr(ptr: number) {
-            let vv = ptr >> pxt.appTarget.compile.vtableShift
+        export function encodeVTPtr(ptr: number, opts: CompileOptions) {
```

Comment:
Is this function never used?

Reply:
Maybe used in another branch by Michal.

Reply:
Looks like it's used in emitter/backvm.ts